src/content/documentation/develop/test-permission-requests.md

```diff
@@ -15,7 +15,7 @@ date: 2019-03-21 12:27:40
 
 # Test permission requests
 
-Your extension may contain two types of permission request: install time and runtime permission requests. This page explains how you can test the way your users will see requests for these permissions.
+Your extension may contain two types of permission request: <dfn>install time</dfn> and <dfn>runtime</dfn> permission requests. This page explains how you can test the way your users will see requests for these permissions.
```

Comment:
This use of `<dfn>` is not quite right here because the parent element doesn't contain the definition of those terms:

> The HTML Definition element (`<dfn>`) is used to indicate the term being defined within the context of a definition phrase or sentence. The `<p>` element, the `<dt>`/`<dd>` pairing, or the `<section>` element which is the nearest ancestor of the `<dfn>` is considered to be the definition of the term.

Reply:
Oh!  Is that from the spec?  Or a Mozilla style guide rule?

Reply:
Why did I ask when I can look it up? :P  That is from the spec.  

Makes me sheepish.  I think `dfn` is one of the most underappreciated elements for semantic markup…and I used it wrong!  :o

Alas.  Will fix shortly.

Reply:
Changed in 4b72086.